releases.md

```diff
@@ -0,0 +1,38 @@
+
+In general, commits will land in master. Once landed, they are cherry picked out of master to each of the stable/LTS branches. Cherry-picked commits will "trickle down" through stable branches moving from *newer* branches to *older* (that is, assuming a `master` that's at `v6.x.x`, commits that land in `master` would cherry pick into `v5.x`; commits into `v5.x` would cherry pick into `v4.x`; and so on). This is opposed to the current joyent/node approach of landing first in the *oldest* and then moving forward towards master.
+
+It's important to note that *not all* commits will land in `master`. Fixes will land first in the *most recent* branch in which the issue is manifest, and will be cherry-picked back to the older branches.
+
+Semver breaking critical security fixes are handled as special cases. Since the major cannot be bumped in either the stable or LTS branches, breaking security fixes *will land as semver-minor bumps* with a *mandatory* documentation requirement. This isn't ideal, but there's really no other option.
+
+Once a stable branch moves into LTS, the documented LTS plan (http://github.com/nodejs/LTS) takes over and oversight of the LTS branch falls under the responsibility of the LTS working group.
+
+## Landing V8 in master
+
+*Stable* V8 releases will land in `master` as frequently as is feasible. The current V8 release schedule has a new stable being cut every six weeks. The idea then, would be to have this stable being pushed into `master` every six weeks. There are, however, a number of practical limitations that could alter the schedule.
+
+Because the six week V8 release cycle does not line up pefectly with the the six month stable branch detailed above, there will be times when a new stable V8 is released *too close* to when the new stable branch would be cut off `master`. In such cases, landing the new stable V8 will be pushed off until immediately *after* the stable branch cut. This is to ensure that there is plenty of time allowed for testing the new stable V8 and ensuring that breaking changes are known and well reviewed before moving into stable.
+
+It is possible for *semver-major* and *semver-minor* updates to V8 to be cherry picked to stable branches, but in general, *no breaking V8 changes* can land in stable branches. This means that when the annual LTS release is cut, the version of V8 included will have had at least six months of time to stablize.
+
+## Versioning
+
+The Node.js version will be primarily rooted in the *stable* branches. Every *stable* branch represents one *semver-major*. Releases from `master` are essentially *alpha* *pre-release* versions of the next master.
+
+For instance, suppose the *next* upcoming stable will be `v5.0.0`. The *alpha* releases cut from `master` would be tagged using a version like `v5.0.0-next-x`, where `x` is a monotonically increasing release number.
```

Comment:
`v5.0.0-next.x`

"is a monotonically increasing release number for the current major version in `master`".